index.bs

```diff
@@ -1064,6 +1067,12 @@ associated.
     ::  This member describes the [=[RPS]=]' requirements regarding availability of the [=Client-side-resident Credential 
         Private Key=]. If the parameter is set to true, the authenticator MUST create a 
         [=Client-side-resident Credential Private Key=] when creating a [=public key credential=].
+
+    :   <dfn>requireUserVerification</dfn>
+    ::  If this parameter is set to <i>true</i>, only authenticators capable of user verification would participate in the 
+        {{CredentialsContainer/create()}} operation, thereby ensuring the authenticator will perform the user verification
```

Comment:
It's not enough to just perform [user verification](https://s3.amazonaws.com/pr-preview/w3c/webauthn/acz.html#user-verification). Even if the authenticator can "distinguish individual users", it needs to record which user authorized create() and only sign credentials when the same user authorizes a get() call.

I'd like to use a more precise name than `requireUserVerification` for this bit. `constrainGetToSameUser` captures the idea, but there may be a shorter expression of it.